console/src/main/scripts/plugins/metrics/ts/alertsCenterList.ts

```diff
@@ -57,16 +57,9 @@ module HawkularMetrics {
                 private $location:ng.ILocationService) {
       $scope.ac = this;
 
-      if ($routeParams.timeOffset === undefined) {
-        $routeParams.timeOffset = 3600000 * 12;
-        $scope.hkParams.timeOffset = $routeParams.timeOffset;
-      }
-
-      this.alertsTimeOffset = $routeParams.timeOffset;
-
       // If the end time is not specified in URL use current time as end time
       this.alertsTimeEnd = $routeParams.endTime ? $routeParams.endTime : Date.now();
-      this.alertsTimeStart = this.alertsTimeEnd - this.alertsTimeOffset;
+      this.alertsTimeStart = this.alertsTimeEnd - HawkularNav.getTimeOffset();
```

Comment:
I have some doubts with this PR. It looks to bypass the url offset parameters which give each screen its stateless RESTful url architecture. Each screens data keys should be available from the url and stored somewhere.